templates/notes_app_shelf/TEMPLATE.md

```diff
@@ -0,0 +1,126 @@
+---
+name: Flutter + Firebase Auth + Shelf Backend
+description: Build a Notes App using Flutter Web, Firebase Auth and a custom backend using Shelf.
+tags: ["firebase", "shelf", "flutter", "firestore"]
+username: Invertase
+---
+
+# Flutter + Firebase Auth + Shelf Backend
+
+## Overview
+
+Build a Simple Notes App using Flutter Web, Firebase Auth and Dart for our backend.
+
+### Features

```

Comment:
worth renaming this to something else, because below is not a list of features, but a list of tools

Reply:
I've removed it now. These components are already mentioned in template name & tags so prolly redundant to have it under features again.